.github/workflows/update.yml

```diff
@@ -16,9 +16,6 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Install Foundry
-        uses: foundry-rs/foundry-toolchain@v1
```

Comment:
Maybe we can use https://github.com/bgd-labs/foundry-zksync-toolchain if we need ?

Reply:
I'd rather not rely on it and instead have our own action.